app/models/assignment_invitation.rb

```diff
@@ -34,6 +34,10 @@ def redeem_for(invitee)
     assignment_repo = AssignmentRepo.find_by(assignment: assignment, user: invitee)
     return AssignmentRepo::Creator::Result.success(assignment_repo) if assignment_repo.present?
 
+    unless assignment.invitations_are_enabled?
```

Comment:
Totally a style nit, but what do you think about delegating `Assignment#invitations_enabled?` to `AssignmentInvitations#enabled?`.

Just a little nicer DSL